nmostesting/IS05Utils.py

```diff
@@ -535,6 +535,10 @@ def check_sdp_matches_params(self, portId):
                 sdp_global = sdp_sections[0]
                 sdp_media_sections = sdp_sections[1:]
                 sdp_groups_line = re.search(r"a=group:DUP (.+)", sdp_global)
+                connection_global = None
+                connection_global_line = re.search(r"c=IN IP[4,6] ([^/\r\n]*)(?:/[0-9]+){0,2}", sdp_global)
+                if connection_global_line is not None:
+                    connection_global = connection_global_line.group(1)
```

Comment:
I would suggest getting rid of the `connection_global` temp variable in the interest of compactness and understandability.
```suggestion
                connection_global_line = re.search(r"c=IN IP[4,6] ([^/\r\n]*)(?:/[0-9]+){0,2}", sdp_global)
```